airqo-platform/AirQo-frontend
# MOB-164 Mobile Dynamic Health Tips

#### Summary of Changes (What does this PR do?)

- Making Health Tips Dynamic

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Checkout this branch
- Run the app and checkout the insights Page.
- You should be able to see the health tips 

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [MOB-195](https://airqoteam.atlassian.net/browse/MOB-195) 

#### Screenshots (optional)


[MOB-195]: https://airqoteam.atlassian.net/browse/MOB-195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
